src/components/MesUtilisateurs/InviterUnUtilisateur.tsx

```diff
@@ -100,32 +114,53 @@ export default function InviterUnUtilisateur({
           <span className="color-red">
             *
           </span>
-        </TextInput>
-        <legend
-          aria-describedby="champsObligatoires"
-          className="fr-mb-2w"
-        >
-          Quel rôle souhaitez-vous lui attribuer ?
-          {' '}
-          <span className="color-red">
-            *
-          </span>
-        </legend>
-        <RadioGroup
-          nomGroupe="attributionRole"
-          options={gestionnaires}
-        />
-        <TextInput
-          id={structureId}
-          name="structure"
-          required={true}
-        >
-          Structure
-          {' '}
-          <span className="color-red">
-            *
+          <span className="fr-hint-text">
+            Une invitation lui sera envoyée par mail
           </span>
         </TextInput>
+        {
+          gestionnaires.length > 1 ?
+            <legend
+              aria-describedby="champsObligatoires"
+              className="fr-mb-2w"
+            >
+              Quel rôle souhaitez-vous lui attribuer ?
+              {' '}
+              <span className="color-red">
+                *
+              </span>
+            </legend> :
+            <p className="fr-mb-1w">
+              Rôle attribué à cet utilisateur :
+            </p>
+        }
+        {
+          gestionnaires.length > 1 ?
+            <RadioGroup
+              nomGroupe="attributionRole"
+              onChange={(event) => {
+                setRoleSelectionne(event.target.value)
+              }}
+              options={gestionnaires}
+            /> :
+            <Badge color="purple-glycine">
+              {gestionnaires[0]?.label}
+            </Badge>
+        }
+        {
+          isStructureDisplayed() ?
+            <TextInput
+              id={structureId}
+              name="structure"
+              required={true}
+            >
+              Structure
+              {' '}
```

Comment:
```suggestion
              {'Structure'}
```
Je propose que l'on commence à faire ça qui semble un bon tradeoff. A la limite, il faudrait faire une PR à côté pour corriger partout.